Documentation/cmdref/cilium-agent.md

```diff
@@ -61,6 +61,7 @@ cilium-agent
       --logstash-agent string                       Logstash agent address (default "127.0.0.1:8080")
       --logstash-probe-timer uint32                 Logstash probe timer (seconds) (default 10)
       --masquerade                                  Masquerade packets from endpoints leaving the host (default true)
+      --mtu int                                     Overwrite auto-detected MTU of underlying network (default 1500)
```

Comment:
I got really confused. The default value will not be "1500", it might depend on the user interface's MTU

Reply:
It will be correct when the user actually runs `--help`. This is only the auto generated documentation. I find it very useful that a user can run `cilium-agent --help` and see the value that will be automatically derived.

Reply:
Somehow, for some reason, I thought this would be the actual output of the `--help`